build.gradle

```diff
@@ -62,26 +32,24 @@ gradlePlugin {
 repositories {
   mavenCentral()
   google()
-  jcenter()
 }
 
 dependencies {
   implementation localGroovy()
-  implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion"
   implementation gradleApi()
-  api "guru.nidi:graphviz-java:0.8.1"
+  api "guru.nidi:graphviz-java:0.18.1"
```

Comment:
Can we get this merged at least to resolve #144  ???

Reply:
I opened one here https://github.com/vanniktech/gradle-dependency-graph-generator-plugin/pull/145